entrypoint.sh

```diff
@@ -59,13 +73,13 @@ fi
 # https://groups.google.com/forum/#!topic/airbnb_airflow/4ZGWUzKkBbw
 if check_set "${ENABLE_AIRFLOW_INITDB}"; then
   echo "Initializing database for Airflow..."
-  gosu "${AIRFLOW_USER}" airflow initdb || gosu "${AIRFLOW_USER}" airflow db init
+  gosu "${AIRFLOW_USER}" airflow db init
   echo "Database is initialized with Airflow metadata!"
 fi
 
 if check_set "${ENABLE_AIRFLOW_UPGRADEDB}"; then
   echo "Upgrading database schema for Airflow..."
-  gosu "${AIRFLOW_USER}" airflow upgradedb || gosu "${AIRFLOW_USER}" airflow db upgrade
+  gosu "${AIRFLOW_USER}" airflow db upgrade
   echo "Database is upgraded with latest Airflow metadata schema!"
 fi
```

Comment:
Since the build matrix only builds for Airflow v2, which has since [deprecated `initdb` and `upgradedb`](https://airflow.apache.org/docs/apache-airflow/stable/upgrading-from-1-10/index.html#airflow-cli-changes-in-2-0), therefore I have simplified the command here.